src/components/charts/TotalsPie.tsx

```diff
@@ -43,8 +42,8 @@ export default function TotalsPie({
 
     return guids.map(guid => {
       let total = totals?.[guid] || new Money(0, unit);
-      const account = accounts?.find(a => a.guid === guid) as Account;
-      if (currency && account.commodity.guid !== currency?.guid) {
+      const account = accounts?.find(a => a.guid === guid);
+      if (account && currency && account.commodity.guid !== currency?.guid) {
```

Comment:
for when an account is not found which can happen when importing a different file, initializing, etc.